Resources/translations/SonataUserBundle.de.xliff

```diff
@@ -514,10 +514,6 @@
                 <source>Gender</source>
                 <target>Geschlecht</target>
             </trans-unit>
-            <trans-unit id="Bad credentials">
-                <source>Bad credentials</source>
-                <target>Benutzername oder Passwort ungültig</target>
-            </trans-unit>
```

Comment:
I don't think this translation exist in other translation files. At least I can't find it in other files.